pkg/plugin/v3/scaffolds/internal/templates/config/manager/config.go

```diff
@@ -78,6 +78,18 @@ spec:
         name: manager
         securityContext:
           allowPrivilegeEscalation: false
+        livenessProbe:
+          httpGet:
+            path: /readyz
+            port: 9443
```

Comment:
I think you need to set [`HealthProbeBindAddress`](https://github.com/kubernetes-sigs/controller-runtime/blob/v0.7.0-alpha.6/pkg/manager/manager.go#L212) in `main.go` for this to work.

Reply:
Done in #1856